<?xml version="1.0"?>
<launch>
	<arg name="parameter" default="0.0"/>
	<!-- <arg name="parameter" value="$(arg parameter)"/> -->
	<include file="$(find usb_cam)/launch/usb_cam-test.launch">
	</include>
	
	<node name="sample_data" pkg="slam_car" type="sample_data" 
	output="screen" >
	</node>
	
</launch>
